docs/2_0_ConfigurationGuid.md

```diff
@@ -0,0 +1,5 @@
+### Environment Variables
```

Comment:
There's a typo in the filename 🙂 
I also think the 1.0 configuration guide still applies, possibly with minor tweaks, for what concerns integrating VC AuthN with Keycloak.

Reply:
merged the two files and expanded on one more variable.